__tests__/user/user.query.test.js

```diff
@@ -4,24 +4,58 @@ const client = require('../../utils/apollo-test-client');
 const {
   INVALID_ADMIN_INVITATIONAL_TOKEN,
 } = require('../../error-messages/user.messages');
+const { superAdminUser } = require('./user.variables');
+const { adminLogin, setupApp } = require('../helper-functions');
+const { getAllUsers } = require('../../modules/user/user.service');
+
+jest.mock('../../modules/confirm-email/confirmation-email.service');
 
 require('dotenv').config();
 
 let token;
 let userId;
+let operations;
+
+const testUser = {
```

Comment:
Please, move this to the `user.variables.js` file, you can reuse one user object both for the mutation tests and the query tests

Reply:
solved